router/router.go

```diff
@@ -713,35 +713,13 @@ func (worker *workerT) processDestinationJobs() {
 								sendCtx, cancel := context.WithTimeout(ctx, worker.rt.netClientTimeout)
 								defer cancel()
 								//transformer proxy start
-								if worker.rt.transformerProxy {
-									jobId := destinationJob.JobMetadataArray[0].JobID
-									pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request started`, worker.rt.destName, jobId)
-									rtl_time := time.Now()
-									// Proxy Request Test
-									respStatusCode, respBodyTemp = worker.rt.transformer.ProxyRequestTest(ctx, val, worker.rt.destName, jobId)
-									worker.routerProxyStat.SendTiming(time.Since(rtl_time))
-									pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request ended`, worker.rt.destName, jobId)
-									authType := router_utils.GetAuthType(destinationJob.Destination)
-									if router_utils.IsNotEmptyString(authType) && authType == "OAuth" {
-										pkgLogger.Debugf(`Sending for OAuth destination`)
-										// Token from header of the request
-										respStatusCode, respBodyTemp = worker.rt.HandleOAuthDestResponse(&HandleDestOAuthRespParamsT{
-											ctx:            ctx,
-											destinationJob: destinationJob,
-											workerId:       worker.workerID,
-											trRespStCd:     respStatusCode,
-											trRespBody:     respBodyTemp,
-											secret:         destinationJob.JobMetadataArray[0].Secret,
-										})
-									}
-								} else {
-									rdl_time := time.Now()
-									resp := worker.rt.netHandle.SendPost(sendCtx, val)
-									respStatusCode, respBodyTemp, respContentType = resp.StatusCode, string(resp.ResponseBody), resp.ResponseContentType
-									// stat end
-									worker.routerDeliveryLatencyStat.SendTiming(time.Since(rdl_time))
-								}
-								// transformer proxy end
+								rdl_time := time.Now()
+								jobId := destinationJob.JobMetadataArray[0].JobID
+								resp := worker.rt.netHandle.SendPost(sendCtx, val, worker.rt.destName, jobId, worker.rt.transformerProxy)
```

Comment:
I believe there are no changes required to this method. 
When the proxy test configuration is enabled, rather than enabling this path, can we enable the destination delivery path itself but inform it of the configuration value?